apps/fabric-website/src/components/Nav/Nav.tsx

```diff
@@ -303,6 +311,20 @@ function _hasActiveChild(page: INavPage): boolean {
           if (_isPageActive(grandchildPage)) {
             hasActiveChild = true;
           }
+          if (grandchildPage.pages) {
```

Comment:
I don't really see the reason why there are two separate functions here _hasActiveChild vs _hasActiveGrandchild where one can handle only up to 4 levels and one can handle arbitrary levels. Can we just combine these two functions?